activestorage/app/models/active_storage/blob.rb

```diff
@@ -142,13 +145,14 @@ def service_headers_for_direct_upload
   #
   # Prior to uploading, we compute the checksum, which is sent to the service for transit integrity validation. If the
   # checksum does not match what the service receives, an exception will be raised. We also measure the size of the +io+
-  # and store that in +byte_size+ on the blob record.
+  # and store that in +byte_size+ on the blob record. The content type is automatically extracted from the +io+ unless
+  # the user has specified a +content_type+ and set +identify+ as false.
```

Comment:
Use “you specify” instead of “the user has specified” to match the adjacent paragraphs, which use the second person.

“Set `identify` as false” doesn’t read quite right. How about “pass `identify: false`”?